website/lib/get-all-articles.ts

```diff
@@ -50,6 +50,12 @@ async function readMeta(dir: string, file: string): Promise<MetaWithLink> {
       .use(stringify);
     const vFile = await processor.process(raw);
     const parsed = vFile.data.frontmatter as Meta;
+    if (parsed.title.length > 70) {
```

Comment:
Title should be less or equal to 60? Also keep in mind titleSuffix https://github.com/the-guild-org/docs/blob/c39c2173db2d67eb0c96ae34406d79c8a9b1db89/packages/components/src/define-config.tsx#L77

so ` - The Guild` will be appended too, so your check should contain 60-12=48 characters

Reply:
The <h1> should be between 20 - 70 characters according to SEO expect.

https://neilpatel.com/blog/h1-tag/#:~:text=H%20tags%20are%20important%20for,between%2020%20and%2070%20characters.

Reply:
Your PR title is confusing, I was thing about page title in head